qt/aqt/browser/find_and_replace.py

```diff
@@ -97,6 +133,9 @@ def accept(self) -> None:
         save_is_checked(self.form.re, self.COMBO_NAME + "Regex")
         save_is_checked(self.form.ignoreCase, self.COMBO_NAME + "ignoreCase")
 
+        if not self.form.selected_notes.isChecked():
+            self.note_ids = self.mw.col.find_notes("")
```

Comment:
Is that all right? We could handle the empty id list case on the backend, but that would require much more than two lines.

Reply:
It's potentially hundreds of thousand of note ids we'd have to serialize back and forth, and doing it in Rust should be roughly as simple (eg checking if nids empty in the backend method, and filling it with search_notes_unordered(""))

Reply:
Ah, of course, that makes sense. 🙂

Reply:
One problem, though: The tooltip is showing "123 of 0 Notes updated" that way. Not sure, at which place to fix this.

Reply:
When you call .success() on a CollectionOp with an existing success handler, you can replace the handler with your own. So you could do something like:

```python
op = find_and_replace(parent=self.parentWidget(), ...)
if all_notes:
  op.success(lambda: False)
op.run_in_background()
```

optionally replacing the False with a different message